### PR TITLE
Added usage of annotations for kernel clients

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Issue
https://github.com/networkservicemesh/deployments-k8s/issues/2009

### CODE REVIEW

**Review by glazychev-art — APPROVED** [review]

File: examples/features/ipv6/Kernel2Wireguard2Kernel/README.md [hunk 1]
```diff
@@ -91,15 +92,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-kernel -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/features/ipv6/Kernel2Wireguard2Kernel/README.md:95]: Why is the order changed?

Note: previously we waited for NSC first.

File: examples/features/ipv6/Kernel2Wireguard2Memif/README.md [hunk 2]
```diff
@@ -91,15 +94,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-memif -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/features/ipv6/Kernel2Wireguard2Memif/README.md:97]: Why is order changed?

File: examples/features/ipv6/Kernel2Wireguard2Memif/README.md [hunk 3]
```diff
@@ -28,37 +28,40 @@ kind: Kustomization
...
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder-ip/nsm-1
-
-      nodeSelector:
-        kubernetes.io/hostname: ${NODES[0]}
+  containers:
+  - name: alpine
+    image: alpine
```
Comment (denis-tingaikin) [inline examples/features/ipv6/Kernel2Wireguard2Memif/README.md:56]: Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).

File: examples/use-cases/Kernel2Kernel/README.md [hunk 4]
```diff
@@ -31,34 +31,37 @@ kind: Kustomization
...
-      containers:
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
-      nodeSelector:
-        kubernetes.io/hostname: ${NODE}
+  containers:
+  - name: alpine
+    image: alpine
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Kernel/README.md:59]: Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).

File: examples/use-cases/Kernel2Kernel/README.md [hunk 5]
```diff
@@ -90,15 +93,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-kernel -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Kernel/README.md:96]: Why is the order changed?

File: examples/use-cases/Kernel2Memif/README.md [hunk 6]
```diff
@@ -32,34 +32,37 @@ kind: Kustomization
...
-      containers:
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
-      nodeSelector:
-        kubernetes.io/hostname: ${NODE}
+  containers:
+  - name: alpine
+    image: alpine
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Memif/README.md:60]: Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).

File: examples/use-cases/Kernel2Memif/README.md [hunk 7]
```diff
@@ -91,15 +94,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-memif -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Memif/README.md:97]: Why is the order changed?

File: examples/use-cases/Kernel2Vxlan2Kernel/README.md [hunk 8]
```diff
@@ -31,38 +31,40 @@ kind: Kustomization
...
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
-
-      nodeSelector:
-        kubernetes.io/hostname: ${NODES[0]}
+  containers:
+  - name: alpine
+    image: alpine
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Vxlan2Kernel/README.md:59]: Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).

File: examples/use-cases/Kernel2Vxlan2Kernel/README.md [hunk 9]
```diff
@@ -91,15 +93,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-kernel -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Vxlan2Kernel/README.md:96]: Why is the order changed?

File: examples/use-cases/Kernel2Vxlan2Memif/README.md [hunk 10]
```diff
@@ -32,38 +32,40 @@ kind: Kustomization
...
-        - name: nsc
-          env:
-            - name: NSM_NETWORK_SERVICES
-              value: kernel://icmp-responder/nsm-1
-
-      nodeSelector:
-        kubernetes.io/hostname: ${NODES[0]}
+  containers:
+  - name: alpine
+    image: alpine
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Vxlan2Memif/README.md:60]: Please use a concrete alpine image.
Also, add the alpine image into [external-images.yaml](https://github.com/networkservicemesh/deployments-k8s/blob/main/external-images.yaml).

File: examples/use-cases/Kernel2Vxlan2Memif/README.md [hunk 11]
```diff
@@ -92,15 +94,15 @@ kubectl apply -k .
 
 Wait for applications ready:
 ```bash
-kubectl wait --for=condition=ready --timeout=1m pod -l app=nsc-kernel -n ${NAMESPACE}
+kubectl wait --for=condition=ready --timeout=1m pod -l app=nse-memif -n ${NAMESPACE}
```
Comment (denis-tingaikin) [inline examples/use-cases/Kernel2Vxlan2Memif/README.md:97]: Why is the order changed?

**Comment by Mixaster995** [comment]
@denis-tingaikin fixed review comments

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**